src/openmct-yamcs.js

```diff
@@ -76,7 +76,7 @@ export default function install(
 
         openmct.faults.addProvider(new YamcsFaultProvider(openmct,
             {
-                faultModelConvertor,
+                convertDataToFaultModel,
```

Comment:
Looks like a mismatch of parameters provided to the exoectation here: https://github.com/akhenry/openmct-yamcs/pull/481/files#diff-e3379e1f900a4b0e58c3057f59da360a685831b58d75bf3b80d0bfdaeda4ba88R6

Reply:
wow, really good catch